src/main/data/vcf.js

```diff
@@ -41,12 +43,28 @@ function extractLocusLine(vcfLine: string): LocusLine {
 
 function extractVariant(vcfLine: string): Variant {
   var parts = vcfLine.split('\t');
+  var frequency = null;
+  if (parts.length>=7){
+    var params = parts[7].split(';');
+    for (var i=0;i<params.length;i++) {
+      var param = params[i];
+      if (param.startsWith("AF=")) {
+        frequency = 0.0;
+        var frequenciesStrings = param.substr(3).split(",");
+        for (var j=0;j<frequenciesStrings.length;j++) {
+          frequency = Math.max(frequency,parseFloat(frequenciesStrings[j]));
+        }
+      }
+    }
+  }
 
   return {
     contig: parts[0],
     position: Number(parts[1]),
+    id: parts[2],
     ref: parts[3],
     alt: parts[4],
+    significantFrequency: frequency,
```

Comment:
The reason is that single vcf entry can contain information about more than one gene variant:
``
20 1110696 rs6040355 A G,T 67 PASS NS=2;DP=10;AF=0.333,0.667;AA=T;DB
``
In such situation we need to decide which allel frequency value use for the visualization.
I know about two strategies, but there might be more: 
* take the smalles
* take the biggest (this one is currently implemented)

The alternative to `significantFrequency` field is to store all the 'allel frequency' values and allow user to decide. But again we have to agree about the possible strategies.